### ISSUE TITLE
NVDA fails to run and causes WerFault exceptions

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2872]
**Reported by julianharty on 2012-12-17 12:06**
Neither NVDA portable v0.5 or NVDA 2012.3.1 run on the Windows 7 corporate computer I am using. 

Both were the current, recommended downloads from the NVDA project site. 

For NVDA Portable, the UI starts and runs for a second or two. Then it disappears. I will attach the nvda_debug.log file which is generated. nvda_stderr.log is created, but empty.

For the installed version of NVDA, When the UI starts (by clicking on NVDA.exe) the UI offers me the option to choose the NVDA key. I leave the default value and continue. I then start getting WerFault.exe popups. I noticed that Windows Explorer is no longer visible and cannot start Task Manager. I can use the web browser, and Ctrl+Alt+Del still work. I have included the error message from the Windows Application Log below (I removed some confidential details, which are replaced with the word 'removed'). I will also attach nvda.log which is generated when NVDA starts.

Note: I first tried to run the portable version. I then installed the full program after the portable version didn't work. The problems appear consistently.

I'm a sighted user, using NVDA for testing web accessibility. I only have access to the machine I'm using on an occasional basis.

Here is the extract from the Windows Application Log file:

Log Name:      Application
Source:        Symantec AntiVirus
Date:          17/12/2012 11:41:02
Event ID:      45
Task Category: None
Level:         Error
Keywords:      Classic
User:          removed
Computer:      removed
Description:

SYMANTEC TAMPER PROTECTION ALERT

Target:  C:\Program Files (x86)\Symantec\Symantec Endpoint Protection\12.1.671.4971.105\Bin\ccSvcHst.exe
Event Info:  Create Process
ActionTaken:  Logged
Actor Process:  C:\PROGRAM FILES (X86)\NVDA\NVDA.EXE (PID 2360)
Time:  17 December 2012  11:41:02
Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Symantec AntiVirus" />
    <EventID Qualifiers="49407">45</EventID>
    <Level>2</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2012-12-17T11:41:02.000000000Z" />
    <EventRecordID>8621</EventRecordID>
    <Channel>Application</Channel>
    <Computer>removed</Computer>
    <Security UserID=removed />
  </System>
  <EventData>
    <Data>

SYMANTEC TAMPER PROTECTION ALERT

Target:  C:\Program Files (x86)\Symantec\Symantec Endpoint Protection\12.1.671.4971.105\Bin\ccSvcHst.exe
Event Info:  Create Process
ActionTaken:  Logged
Actor Process:  C:\PROGRAM FILES (X86)\NVDA\NVDA.EXE (PID 2360)
Time:  17 December 2012  11:41:02</Data>
  </EventData>
</Event>

**Comment by bhavyashah** [comment]
Could someone please take a look at the logs attached in https://github.com/nvaccess/nvda/issues/2872#issuecomment-155303216 and https://github.com/nvaccess/nvda/issues/2872#issuecomment-155303217 and let us know if it provides any useful information that might help us independently troubleshoot this issue (since the original author of this ticket can no longer be contacted)?

**Comment by Adriani90** [comment]
@julianharty are you still able to reproduce this? I guess NVDA was blocked by symantec security software or something like that. I am prety sure this is not an NVDA bug.

**Comment by ehollig** [comment]
I believe Symantec was blocking NVDA, or parts of NVDA for some reason. I am going to close this issue, as it is not an NVDA issue, but if it is still occurring, we can reopen.
